buildenv/jenkins/omrbuild.groovy

```diff
@@ -264,7 +264,7 @@ def SPECS = [
         'junitPublish' : true
     ],
     'win_x86-64' : [
-        'label' : 'Windows && x86',
+        'label' : 'compile:windows',
```

Comment:
At some point in the near future we're going to have to distinguish between Windows on x86 and Windows on AArch64. Using your lettering convention above, does that make this `xwindows` ?